packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -130,29 +117,25 @@ const Tabs = ({
     >
       <ul
         role="tablist"
-        className={classNames('Tablist', {
-          'Tablist--full-width': variant === 'full-width'
-        })}
```

Comment:
Why is this being removed?

Reply:
The styling is very minimum for the 'full-width' variant, and we need users to set their own width of Tablist and TabPanel anyway. So I decided to just leave it to the users.